src/main/java/gregtech/common/metatileentities/multi/multiblockpart/MetaTileEntityFuelRodExportHatch.java

```diff
@@ -0,0 +1,112 @@
+package gregtech.common.metatileentities.multi.multiblockpart;
+
+import gregtech.api.capability.IControllable;
+import gregtech.api.gui.GuiTextures;
+import gregtech.api.gui.ModularUI;
+import gregtech.api.gui.widgets.SlotWidget;
+import gregtech.api.metatileentity.MetaTileEntity;
+import gregtech.api.metatileentity.interfaces.IGregTechTileEntity;
+import gregtech.api.metatileentity.multiblock.IFissionReactorHatch;
+import gregtech.api.metatileentity.multiblock.IMultiblockAbilityPart;
+import gregtech.api.metatileentity.multiblock.MultiblockAbility;
+import gregtech.client.renderer.texture.Textures;
+import gregtech.client.renderer.texture.cube.SimpleOverlayRenderer;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.items.IItemHandlerModifiable;
+import net.minecraftforge.items.ItemStackHandler;
+
+import codechicken.lib.render.CCRenderState;
+import codechicken.lib.render.pipeline.IVertexOperation;
+import codechicken.lib.vec.Matrix4;
+
+import java.util.List;
+
+public class MetaTileEntityFuelRodExportHatch extends MetaTileEntityMultiblockNotifiablePart
```

Comment:
Perhaps it could, but there are enough differences between the classes, especially coming down to all of the ILockableHandler implementation details in the import bus, that it's also plausible to just separate the two out (and not deal with 16 different if statements in each function).